e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/embed-parameters.cy.spec.ts

```diff
@@ -116,107 +134,125 @@ describe("scenarios > embedding > sdk iframe embed setup > embed parameters", ()
         .findByTestId("dashboard-parameters-widget-container")
         .should("not.exist");
 
+      H.expectUnstructuredSnowplowEvent(
+        {
+          event: "embed_wizard_option_changed",
+          event_detail: "hiddenParameters",
+        },
+        2,
+      );
+
       cy.log("code snippet should contain the hidden parameters");
       getEmbedSidebar().within(() => {
         cy.findByText("Get Code").click();
         codeBlock().should("contain", '"hiddenParameters"');
-        codeBlock().should("not.contain", '"initialParameters"');
         codeBlock().should("contain", '"id"');
         codeBlock().should("contain", '"product_id"');
       });
     });
   });
 
-  it("shows no parameters message for dashboards without parameters", () => {
-    navigateToEmbedOptionsStep({
-      experience: "dashboard",
-      resourceName: "Orders in a dashboard",
+  describe("questions with parameters", () => {
```

Comment:
I re-grouped the tests a bit in 3 describe blocks, to make it neater to read:

- questions with parameters
- dashboards with parameters
- resources without parameters